### DISCUSSION TITLE
Intel Ingest / Fields

### DISCUSSION THREAD

**Discussion by xfaith** [Security-Onion-Solutions/securityonion#8247] [General]
So I added the CriticalPathSecurity Zeek Intel feeds to SecurityOnion, and I can see it in Dashboards, and I can see it in Kibana, but does not look like I can see what I wouild believe would be all  the fields from the intel.dat  (are all the fields available already, or do I have to add them somewhere in the ingest path?   (meta.do-notice, meta.source)  Just something to get a little more details in the intel findings.   

Thanks

**Comment by dougburks** [comment]
Have you tried drilling into an individual log to see all the fields?

**Reply by xfaith** [reply]
So I was able to to find one (sources) and use that in the Intel dashboard intel.sources.  Which I believe adds the meta.source.
As for the other ones .  I did review the intel.log from Zeek and does not look like meta.do_notice or meta_desc are included.  Might have to circle back onto the intel feeds, as I was reading adding:

@load policy/integration/collective-intel to your local.zeek should add the meta stuff.  Hmm possibly back to the drawing board on trying to figure out why its not working.

Edit:  Actually I think its the @load policy/frameworks/intel/do_notice.

**Reply by dougburks** [reply]
Here is the Elasticsearch ingest parser for Zeek intel:
https://github.com/Security-Onion-Solutions/securityonion/blob/master/salt/elasticsearch/files/ingest/zeek.intel

Here you can see that `sources` is renamed to `intel.sources`:
https://github.com/Security-Onion-Solutions/securityonion/blob/master/salt/elasticsearch/files/ingest/zeek.intel#L15

meta.do_notice is just a boolean value that controls whether a separate notice is generated, so it doesn't actually get logged into intel.log.

**Reply by xfaith** [reply]
Got it, and thanks for the additional details.   I am showing a part from the CriticalPathSecurity Intel Feed: 

I was trying to see if that BOTNET column was ingested to be viewable somewhere. 

#fields	indicator	indicator_type	meta.source	meta.do_notice	meta.desc
51.178.161.32	Intel::ADDR	ABUSE-CH	F	BOTNET

Not in front of machine, but maybe I should be doing a search for file.description?  
  { "rename": 	{ "field": "message2.file_desc", 	"target_field": "file.description",	"ignore_missing": true 	} },

**Reply by dougburks** [reply]
I think `file_desc` is not the same as `meta.desc`. 

Take a look at the `message` field which shows the full Zeek log and see if it shows `meta.desc`.
